### PR TITLE
sFraxEth deployment script

### PR DESCRIPTION
**Pull request by julianmrodri**
StakedFraxEth Deployment and verification scripts

### CODE REVIEW

File: scripts/deployment/phase2-assets/collaterals/deploy_sfrax_eth.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      chainlinkFeed: networkConfig[chainId].chainlinkFeeds.ETH,
+      oracleError: oracleError.toString(),
+      erc20: networkConfig[chainId].tokens.sfrxETH,
+      maxTradeVolume: fp('1e6').toString(), // $1m,
+      oracleTimeout: '3600', // 1 hr
+      targetName: hre.ethers.utils.formatBytes32String('ETH'),
+      defaultThreshold: fp('0.02').add(oracleError).toString(), // ~2.5%
+      delayUntilDefault: bn('86400').toString(), // 24h
+    },
+    '0', // revenueHiding = 0
```
Comment (tbrent) [inline scripts/deployment/phase2-assets/collaterals/deploy_sfrax_eth.ts:63]: let's use `fp('1e-6')` like we do elsewhere when we aren't 100% sure and want to permit small rounding errors

File: scripts/verification/collateral-plugins/verify_sfrax_eth.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        chainlinkFeed: networkConfig[chainId].chainlinkFeeds.ETH,
+        oracleError: oracleError.toString(),
+        erc20: networkConfig[chainId].tokens.sfrxETH,
+        maxTradeVolume: fp('1e6').toString(), // $1m,
+        oracleTimeout: oracleTimeout(chainId, '3600').toString(), // 1 hr
+        targetName: hre.ethers.utils.formatBytes32String('ETH'),
+        defaultThreshold: fp('0.02').add(oracleError).toString(), // ~2.5%
+        delayUntilDefault: bn('86400').toString(), // 24h
+      },
+      '0', // revenueHiding = 0
```
Comment (tbrent) [inline scripts/verification/collateral-plugins/verify_sfrax_eth.ts:46]: this would have to change too

**Review by tbrent — COMMENTED** [review]
LGTM except for revenueHiding bump

File: scripts/deployment/phase2-assets/collaterals/deploy_sfrax_eth.ts (hunk 1 above)
Comment (julianmrodri) [reply scripts/deployment/phase2-assets/collaterals/deploy_sfrax_eth.ts:63]: Good catch. In Reth and Lido we are using `fp('1e-4').toString(),` should we do the same?

Comment (julianmrodri) [reply scripts/deployment/phase2-assets/collaterals/deploy_sfrax_eth.ts:63]: cbETH the same

File: scripts/verification/collateral-plugins/verify_sfrax_eth.ts (hunk 2 above)
Comment (julianmrodri) [reply scripts/verification/collateral-plugins/verify_sfrax_eth.ts]: Setting to `1e-4` as agreed offline

File: scripts/deployment/phase2-assets/collaterals/deploy_sfrax_eth.ts (hunk 1 above)
Comment (julianmrodri) [reply scripts/deployment/phase2-assets/collaterals/deploy_sfrax_eth.ts]: Setting to `1e-4` as agreed offline

**Review by pmckelvy1 — APPROVED** [review]
lgtm

**Pull request merged**